spec/controllers/admin/users_controller_spec.rb

```diff
@@ -215,6 +248,40 @@
         expect(flash[:error]).to match(/Access denied/)
       end
     end
+
+    describe 'user' do
```

Comment:
I think it's a pretty good idea to use this describe blocks to structure the tests by HTTP action.
e.g. describe 'POST update' ....